wdio.conf.js

```diff
@@ -78,8 +78,8 @@ exports.config = {
                 '--disable-infobars',
                 '--headless',
                 '--disable-gpu',
-                '--window-size=1920,1080'] : [])
-        }
+                '--window-size=1920,1080'] : []),
+        },
```

Comment:
are these formatting changes coming from your local vs code configuration or it's a global configuration that didn't work for one of us? I think we should align on this to avoid them happening again for other files.

Reply:
I think its from the newer pie-eslint package being installed and other people not applying the lint rules on save :)